packages/nextjs/src/components/_common/kontakt-oss-kanal/_shared-utils/icon/Icon.module.scss

```
@@ -2,14 +2,14 @@
     align-items: center;
     align-self: flex-start;
     background-color: var(--a-white);
-    border-radius: 2.75rem;
+    border-radius: var(--a-spacing-11);
```

Dette synes jeg er en mer robust måte å sikre full border-radius, uansett størrelse på sirkelen.
```suggestion
    border-radius: 999px;
```